src/main/java/seedu/bigpp/component/cpu/CPU.java

```diff
@@ -73,16 +74,14 @@ public void setSocket(String socket) {
 
     @Override
     public String toString() {
-        return "CPU{" +
-                "name='" + super.getName() + '\'' +
-                ", brand='" + super.getBrand() + '\'' +
-                ", price=" + super.getPrice() +
-                ", cores=" + cores +
-                ", threads=" + threads +
-                ", baseClock=" + baseClock +
-                ", boostClock=" + boostClock +
-                ", power=" + power +
-                ", socket='" + socket + '\'' +
-                '}';
+        return  "NAME: " + super.getName() +
+                "\nBRAND: " + super.getBrand() +
+                "\nPRICE: " + super.getPrice() +
+                "\nCORES: " + cores +
+                "\nTHREADS: " + threads +
+                "\nBASECLOCK: " + baseClock +
+                "\nBOOSTCLOCK: " + boostClock +
+                "\nPOWER: " + power +
+                "\nSOCKET: " + socket;
```

Comment:
nice way to format the printing, however in the future we might add flags to print extra information like these